app/menu/menu.tsx

```diff
@@ -94,7 +94,14 @@ export default class MenuComponent extends React.Component {
                     !this.props.user?.selectedGroup.ownedDomain &&
                     !this.props.user?.isInDefaultGroup() && (
                       <li onClick={this.dismissMenu.bind(this)}>
-                        <a target="_blank" href="https://buildbuddy.typeform.com/to/PFjD5A">
+                        <a
```

Comment:
i'd add a little `router.navigateToCreateOrg()` helper method in router.tsx like this: https://github.com/buildbuddy-io/buildbuddy/blob/master/app/router/router.tsx#L43

navigating using `<a href=` causes a full page reload, whereas navigating with the router just causes an react re-render

you can have router check capabilities and send you to typeform if necessary or keep that logic here

Reply:
Ah kk, what do you think about adding a `Link` component which automatically does the right thing?

Reply:
let's stick with the router methods for now, we can migrate all router usage in the future if that seems better - but right now we have some logic in there to check if the app is capable to navigating to the destination etc that would make a link component kinda messy

Reply:
Ah I see -- Done! Wasn't sure whether to show the alert in `Router`, LMK if it makes sense